client/src/component/Main.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Switch, Route } from 'react-router-dom' //Switch and Route Library
+import Home from '../pages/home/home.jsx'
+import SignUp from '../pages/signUp/signUp.jsx'	//link to Sign Up page
+import loginRoute from '../pages/login/loginRoute.jsx' //link to Router for login
+
+function Main() {
+  return (
+      <div>
```

Comment:
Use `Fragment`

Reply:
Used the HTML5 semantic `<nav>` instead, because I changed it to there only one element being rendered, not an array of elements.